_source/user-guide/infrastructure-monitoring/custom-metrics.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: article
+title: Custom metrics
+permalink: /user-guide/infrastructure-monitoring/custom-metrics
+flags:
+  admin: true
+  #logzio-plan: pro
+tags:
+  - Grafana
+contributors:
+  - shalper
+---
+
+You can send your application metrics to your Logz.io Infrastructure Monitoring account and benefit from all of the advantages of your tried and true dashboards.
+
+To do this, you'll need to send your application metrics to Logz.io in JSON format and stick to the formatting guidelines.
+
+##### Overview
+{:.no_toc}
+
+1. toc list
+{:toc}
+
+#### JSON format
+{:.no_toc}
+
+Our guiding principle is to use key-value pairs to define metrics and their dimensions.
+Here's the general template.
+
+  ```
+  {
+	"metrics": {
+		"my_first_metric.unit": number,
+		"my_second_metric.unit": number,
+		"my_third_metric.unit": number,
+		"my_fourtn_metrlc.unit": number
+	},
+	"dimensions": {
+		"dimension_l": "value",
+		"dimension_2": "value",
+		"dimension_3": "value",
+		"dimension_4": "value"
+	}
+  }
+  ```
+
+The reason being that we want to send _fewer and longer_ documents. 
+By grouping all possible metrics and dimensions into one document we maximize Elasticsearch's indexing power. In general, the _number_ of documents has a greater impact on Elasticsearch indexing power than document length.
+
+The best way to explain the guidelines is to look at a few examples. So let's get to it.
+
+<div class="tasklist">
+
+##### Single metric with dimensions
+
+In this example, we have one metric per document. We also have a metric field `name`, and its `value` is logged as a dimension.
+
+```
+{
+    "metric": {
+        "name": "refresh_page.duration"
+    },
+    "dimensions": {
+        "environment": "production",
+        "version": "12.05.01",
+        "unit": "milliseconds",
```

Comment:
This example is already quite "rigged" towards our format. 
The most realistic examples are on slides 4 + 5 (you used an example from slide 5 in a great way later on 'what to avoid').

If you do want to keep this example, not that in the original example in the slides, the fields 'version' and 'unit' were out side of the 'dimensions' section